src/read_csv.py

```diff
@@ -3,22 +3,25 @@
 '''
 import urllib.request
 import pandas as pd
-'''
-fuction to fetch data.
-'''
+
 def fetch_data():
+    '''
+    fuction to fetch data.
+    '''
     path = "https://github.com/EKU-Summer-2021/intelligent_system_data/blob/main/" \
     "Intelligent%20System%20Data/KP/KP_10.csv "
     urllib.request.urlretrieve(path, "10.csv")
-'''
-fuction to load data.
-'''
+
 def load_data():
```

Comment:
Instead of loading the data statically, please have a data parameter which contains the path
```
def load_data(data):
    fetch_data()
    return pd.read_csv(data, names=["w", "v"])
```